pkg/activator/throttler.go

```
@@ -234,7 +235,7 @@ func (t *Throttler) updateAllBreakerCapacity() {
 // This function must not be called in parallel to not induce a wrong order of events.
 func (t *Throttler) endpointsUpdated(newObj interface{}) {
 	endpoints := newObj.(*corev1.Endpoints)
-	addresses := resources.ReadyAddressCount(endpoints)
+	addresses, _ := resources.NewFixedEndpointsCounter(endpoints).ReadyCount()
```

See above on ignoring errors.